docs/cli.md

```diff
@@ -94,10 +98,79 @@ With `--all` flag, you create config file including all default setting of offic
       executable_methods:
         - method_name: "node_governance_alarm"`
 ```
-
 Also, You can set the home directory of VATZ (default "~/.vatz") with `--home` flag.
 And you can create the new config file with your disired name with `--output` flag
 
+## Start
+You can start Vatz when you finish set up config default.yaml or your own config yaml file.
+```
+./vatz start --help                 
+start VATZ
+
+Usage:
+   start [flags]
+
+Flags:
+      --config string       VATZ config file. (default "default.yaml")
+  -h, --help                help for start
+      --log string          log file export to.
+      --prometheus string   prometheus port number. (default "18080")
+
+Global Flags:
+      --debug   Enable debug mode on Log.
+      --trace   Enable Trace mode on Log.
+
+```
+You can simply start VATZ with command
+```
+~$ ./vatz start 
+```
+This will start VATZ with default config, that's been created from previous command `vatz init`. <br>
+You can set exact config file with flag --config such as 
+```
+~$ ./vatz start --config /root/User/vatz-config.yaml
+```
+You need to use absolute path. 
+You can also check your own log by its level, adding flag `--debug` or `--trace`.
+
+## Stop
+You can stop(kill gracefully) Vatz when you want to terminate vatz process. 
```

Comment:
@xellos00 
It seems like a good idea to change it to VATZ for word uniformity.

Reply:
Yeah, That's right. I agree!! I will make everything to `VATZ`